src/core/sensor/qgsiodevicesensor.h

```diff
@@ -279,6 +292,8 @@ class CORE_EXPORT QgsSerialPortSensor : public QgsIODeviceSensor
 
     QString mPortName;
 
+    QSerialPort::BaudRate mBaudrate;
```

Comment:
Please rename that to mBaudRate to match the capitalization of the getter/sette.